.github/workflows/coder.yaml

```diff
@@ -276,7 +276,7 @@ jobs:
 
   test-go:
     name: "test/go"
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os == 'ubuntu-latest' && 'buildjet-16vcpu-ubuntu-2204' || matrix.os }}
```

Comment:
Why don't you just change the os in the matrix?

Reply:
actually, it's a required check so i'd rather leave it so we don't pollute job names

Reply:
oh reasonable